README.md

```
@@ -1,33 +1,147 @@
+
+## Services
+- **kernel-keymanager-service**: Core microservice that exposes REST APIs. 
+- **keys-generator**: Utility job used to generate the initial set of cryptographic keys required by MOSIP.
+- **keys-migrator**: Utility tool used to securely migrate cryptographic keys between HSMs.
+
+> **Note**: Use Mosip Auth Adaptor for authentication and authorization to access the Rest APIs.
+
+## Local Setup
+There are two ways to set up the Key Manager service locally:
+1. [Local Deployment](#local-deployment) (Running the JAR file)
+2. [Local Setup using Docker](#local-setup-using-docker-image)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)